src/vector/jitsi/index.ts

```diff
@@ -311,7 +311,7 @@ function closeConference() {
 }
 
 // event handler bound in HTML
-function joinConference(audioDevice?: string, videoDevice?: string) {
+function joinConference(audioOutput?: string, audioInput?: string, videoInput?: string) {
```

Comment:
```suggestion
function joinConference(devices: {audioOutput?: string, audioInput?: string, videoInput?: string}) {
```

Perhaps this (or similar) would have some benefit?